cairo/src/model.cairo

```diff
@@ -213,29 +213,28 @@ namespace model {
 
     // @notice A struct representing the header of an Eth block.
     struct BlockHeader {
-        base_fee_per_gas: felt,
-        blob_gas_used: felt,
-        bloom_len: felt,
-        bloom: felt*,
+        parent_hash: Uint256,
+        ommers_hash: Uint256,
         coinbase: felt,
-        difficulty: felt,
-        excess_blob_gas: felt,
-        extra_data_len: felt,
-        extra_data: felt*,
+        state_root: Uint256,
+        transactions_root: Uint256,
+        receipt_root: Uint256,
+        withdrawals_root: Option,
+        bloom: felt*,
+        difficulty: Uint256,
+        number: felt,
         gas_limit: felt,
         gas_used: felt,
-        hash: Uint256,
+        timestamp: felt,
         mix_hash: Uint256,
         nonce: felt,
-        number: felt,
-        parent_beacon_block_root: Uint256,
-        parent_hash: Uint256,
-        receipt_trie: Uint256,
-        state_root: Uint256,
-        timestamp: felt,
-        transactions_trie: Uint256,
-        uncle_hash: Uint256,
-        withdrawals_root: Uint256,
+        base_fee_per_gas: Option,
+        blob_gas_used: Option,
+        excess_blob_gas: Option,
+        parent_beacon_block_root: Option,
+        requests_root: Option,
+        extra_data_len: felt,
+        extra_data: felt*,
```

Comment:
No `prev_randao: Bytes32` ?

Reply:
prev_randao is mixed hash here as per https://github.com/paradigmxyz/reth/blob/96f1406a14b4e9d9ef4ed22879accd34c89430ec/crates/rpc/rpc-eth-types/src/simulate.rs#L292

Reply:
EIP confirms that is it only a suggested renaming https://eips.ethereum.org/EIPS/eip-4399
It seems clients kept `mixHash` and did not do the renaming
Same in geth https://github.com/ethereum/go-ethereum/blob/65e5ca7d8126f7a8c708f8affb64f16c22cc63c0/core/types/block.go#L74

Reply:
I think that it's because the name "randao" is misleading as some people have believed that it was a safe random number